src/Microsoft.DotNet.Interactive.Mermaid/Microsoft.DotNet.Interactive.Mermaid.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  
+    
+ <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <PackageDescription>Support for dotnet-interactive</PackageDescription>
+    <PackageTags>dotnet-interactive KQL Kusto Data</PackageTags>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <NoWarn>$(NoWarn);NU5100;VSTHRD002</NoWarn><!-- dll outside of lib/ folder -->
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.DotNet.Interactive.Formatting\Microsoft.DotNet.Interactive.Formatting.csproj" />
```

Comment:
Shouldn't need to specify this since it comes in transitively.

Reply:
removed